multi-agent/adapters/inbound/temporal/workflows/scheduled_session_workflow.py

```
@@ -0,0 +1,132 @@
+                        status=outcome,
+                        started_at=started_at,
+                        failure_reason=failure_reason,
+                    ),
+                    start_to_close_timeout=_ACTIVITY_TIMEOUT,
+                    retry_policy=_ACTIVITY_RETRY,
+                )
+
+        if run_id is None:
+            raise RuntimeError(
+                f"Failed to create session for prompt {params.prompt_id}"
+            )
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)